src/pages/workspaces/workspace/notebooks/TerminalLauncher.js

```diff
@@ -101,10 +101,18 @@ const TerminalLauncher = _.flow(
       })
     } else {
       return div({ style: { padding: '2rem' } }, [
-        spinner({ style: { color: colors.primary(), marginRight: '0.5rem' } }),
-        (clusterStatus === 'Creating' || !cluster) ?
-          'Creating runtime environment. You can navigate away and return in 5-10 minutes.' :
-          'Starting runtime environment, this may take up to 2 minutes.'
+        (clusterStatus !== 'Stopped') && spinner({ style: { color: colors.primary(), marginRight: '0.5rem' } }),
+        (() => {
+          if (clusterStatus === 'Creating' || !cluster) {
+            return 'Creating runtime environment. You can navigate away and return in 5-10 minutes.'
+          } else if (clusterStatus === 'Stopping') {
+            return 'Notebook runtime environment is stopping. You can restart it after it finishes.'
+          } else if (clusterStatus === 'Stopped') {
+            return 'Notebook runtime environment is stopped. Start it to edit your notebook.'
+          } else {
```

Comment:
I think we should explicitly check for `'Starting'`, even just for documentation/readability. Also, that will allow us to use the `else` case to handle unexpected status.

Reply:
Follow-up: here are the status values defined in the leonardo source (https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/google/GoogleModel.scala#L55):
```
  case object Unknown extends ClusterStatus
  case object Creating extends ClusterStatus
  case object Running extends ClusterStatus
  case object Updating extends ClusterStatus
  case object Error extends ClusterStatus
  case object Deleting extends ClusterStatus

  // note: the below are Leo-specific statuses, not Dataproc statuses
  case object Deleted extends ClusterStatus
  case object Stopping extends ClusterStatus
  case object Stopped extends ClusterStatus
  case object Starting extends ClusterStatus
```
We should probably be ready for all of these, or at least specifically handle `'Starting'`.